README.md

```diff
@@ -1,2 +1,135 @@
-# CapstOne
-Capstone project
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo https://ferizad.github.io/CapstOne](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# 📖 [JavaScript capstone project] <a name="about-project"></a>
+
+**[JavaScript capstone project]** 
+
+A website for a concert.
+
+## 🛠 Built With <a name="built-with">
+-HTML
+-CSS
+-JS</a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="#">HTML CSS AND JAVASCRIPT</a></li>
+  </ul>
+</details>
+
+### Key Features <a name="key-features"></a>
+
+- **[Linters]**
+  > A linter is a tool that analyzes your source code to flag programming errors, bugs, stylistic errors, and suspicious constructs
+- **[.gitignore]**
+  > Specifies intentionally untracked files to ignore
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live Demo Link](https://ferizad.github.io/CapstOne)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+### Prerequisites
+
+- A text editor (preferably Visual Studio Code)
+- Github
+
+### Setup
+
+Clone this repository to your desired folder:
+
+  cd my-folder
+  git clone https://github.com/Ferizad/CapstOne
+
+### Install
+
+Install this project with:
+
+  -  [Git](https://git-scm.com/downloads)
+  -  [Node](https://nodejs.org/en/download/)
+
+### Usage
+
+### Run tests
+
+### Deployment
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Ferizad**
+
+- GitHub: [@Ferizad](https://github.com/Ferizad)
+- Twitter: [@FerizadK](https://twitter.com/FerizadK)
+- LinkedIn: [Ferizad Kanafina](https://www.linkedin.com/in/ferizad-kanafina-b41061253/)
+
+## 🔭 Future Features <a name="future-features"></a>
+
+
+- [ ] **[Finish the mobile version]**
+- [ ] **[Make some changes]**
+- [ ] **[link the social icons]**
+- [ ] **[Finish the desktop version]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
```

Comment:
- [ ] [optional]It would have been nice if you added a link to the project issues page, for easier contribution to the project